src/ripple/app/ledger/TransactionMaster.h

```diff
@@ -39,7 +39,10 @@ class TransactionMaster
     TransactionMaster& operator= (TransactionMaster const&) = delete;
 
     std::shared_ptr<Transaction>
-    fetch (uint256 const& , bool checkDisk);
+    fetch_from_cache (uint256 const&);
+
+    std::shared_ptr<Transaction>
+    fetch (uint256 const& , int& iError);
```

Comment:
I'd change the type to `error_code_i` and rename the parameter `ec`. (here and in other places). More descriptive param names is usually a good thing, but for error codes `ec` is usually used in the codebase (`boost::asio` uses `ec` too). Side note: We also don't use hungarian notation, so we'd usually leave off the `i` in `iError`.